app/README.md

```diff
@@ -7,7 +7,7 @@ This folder contains all of the react client code for the app
 ```javascript
 let sampleState = {
   // This should be a singular object with the users properties
-  user: { username: 'bob', id: 5 },
+  user: { id: 5, username: 'bob', points: 555, wins: 3, emojicoins: 3, picture: {thumbnail: 'http://cdn.photonesta.com/images/data.whicdn.com/images/127977939/original.png'} }
```

Comment:
Please refactor so that these are on their own line with comments after explaining exactly what each attribute is and what it will be used for